_component_design/choice-tiles.md

```diff
@@ -239,22 +264,22 @@ For selected states, the iconography changes color to match the given state. In
 
 <section class="static-section" markdown="1">
 
-## Handling errors
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Single-select error
+### Disabled
```

Comment:
the visual is definitely not the right one here.

Reply:
That is totally by bad - fixed!